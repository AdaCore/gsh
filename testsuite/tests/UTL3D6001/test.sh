dirname
dirname a b c
dirname ""
dirname "."
dirname /
dirname \\
dirname //
dirname \\\\

dirname toto
dirname //toto
dirname ////
dirname ////toto//
echo B`dirname "/tata "/toto`B
dirname /toto\\tata.txt
