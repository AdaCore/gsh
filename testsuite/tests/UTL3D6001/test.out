[error] dirname: need at least one operand
[error] dirname: does not accept more than one operand
.
.
/
/
//
\\
.
//
/
/
B/tata B
/toto
